addons/point_of_sale/tools/posbox/configuration/posbox_update.sh

```diff
@@ -26,9 +36,4 @@ sudo find / -type f -name "*.iotpatch" 2> /dev/null | while read iotpatch; do
     done
 done
 
-sudo mount -o remount,ro /
-sudo mount -o remount,rw /root_bypass_ramdisks/etc/cups
-
-sudo service led-status start
-
-(sleep 5 && sudo service odoo restart) &
+sudo systemctl restart odoo
```

Comment:
keep the "service resart" because if the old image will be connected to a database in v17 ==> there won't be an "odoo.service"